src/XMakeBuildEngine/Resources/xlf/Strings.cs.xlf

```diff
@@ -811,8 +811,8 @@
         <note />
       </trans-unit>
       <trans-unit id="ProjectMustBeInMSBuildXmlNamespace">
-        <source>MSB4041: The default XML namespace of the project must be the MSBuild XML namespace. If the project is authored in the MSBuild 2003 format, please add xmlns="{0}" to the &lt;Project&gt; element. If the project has been authored in the old 1.0 or 1.2 format, please convert it to MSBuild 2003 format.</source>
-        <target state="translated">MSB4041: Výchozí obor názvů XML projektu musí být oborem názvů XML nástroje MSBuild. Pokud je projekt vytvořen ve formátu MSBuild 2003, přidejte do elementu atribut xmlns="{0}". Jestliže byl projekt vytvořen ve starém formátu 1.0 nebo 1.2, převeďte jej na formát MSBuild 2003.</target>
+        <source>MSB4041: The default XML namespace of the project must be the MSBuild XML namespace or no namspace. If the project is authored in the MSBuild 2003 format, please add xmlns="{0}" to the &lt;Project&gt; element. If the project has been authored in the old 1.0 or 1.2 format, please convert it to MSBuild 2003 format.</source>
```

Comment:
Ooh, can we sneak in a fix to `s/namspace/namespace/` in the English string before doing this?